# Change to build and run the tests on Solaris
# Not suitable for upstream

--- OpenSP-1.5.2/tests/Makefile.am.orig
+++ OpenSP-1.5.2/tests/Makefile.am
@@ -32,11 +32,12 @@
 EXTRA_DIST = $(TESTS_THAT_PASS) $(TESTS_THAT_FAIL) $(DTDDECL_TEST)
 
 
+SHELL = /bin/sh
+
 TESTS_ENVIRONMENT = top_srcdir=$(top_srcdir) \
 	PATH=.:../nsgmls:../sgmlnorm:../sx:../spent:../spcat:../spam:$$PATH \
 	ONSGMLS=`echo onsgmls|sed '$(transform)'`\
-	OSGMLNORM=`echo osgmlnorm|sed '$(transform)'`\
-	$(SHELL)
+	OSGMLNORM=`echo osgmlnorm|sed '$(transform)'`
 
 # Two auxiliary programs used by the tests.
 INCLUDES = -I.. -I$(top_srcdir)/lib -I$(top_srcdir)/intl
